src/ui/digital-exchange/components/common/ComponentInstallActions.js

```diff
@@ -124,6 +129,7 @@ ComponentInstallActions.propTypes = {
   onRecheckStatus: PropTypes.func.isRequired,
   onRetryAction: PropTypes.func.isRequired,
   installUninstallLoading: PropTypes.bool.isRequired,
+  componentUsageList: PropTypes.arrayOf(PropTypes.object).isRequired,
```

Comment:
pls avoid `PropTypes.object`. Try to use `PropTypes.shape` instead (even better with detailed shape of object)